boards/BMS/Src/main.c

```diff
@@ -36,14 +36,11 @@
 #include "Io_SoftwareWatchdog.h"
 #include "Io_Imd.h"
 #include "Io_HeartbeatMonitor.h"
+#include "Io_RgbLedSequence.h"
 
 #include "App_BmsWorld.h"
-#include "App_CanTx.h"
```

Comment:
These headers are by design in `BmsWorld.h` already, so it's redundant to include them